openslides_backend/action/actions/user/forget_password.py

```diff
@@ -41,6 +41,8 @@ def get_updated_instances(self, action_data: ActionData) -> ActionData:
 Please open the following link and choose a new password:
 {url}/login/forget-password-confirm?user_id={user_id}&token={token}
 
+The link will only be valid for 10 minutes.
+
 For completeness your username: {username}"""
```

Comment:
I think this line can be removed then

Reply:
I added that because of https://github.com/OpenSlides/openslides-backend/issues/1979#issuecomment-1829528618 is that retracted?

Reply:
I meant the last line, "For completeness your username: {username}", as that information is now in the subject

Reply:
ah okay, I'll do that then